tests/python/unittest/test_operator.py

```diff
@@ -9286,21 +9286,26 @@ def test_sample_normal_default_shape():
     assert s.shape == (1, 1)
 
 
-def test_min_max_inf():
-    dtypes = [np.float32, np.double]
-    elem_list = [-1, 1, 0, np.inf, -np.inf]
-
+def test_inf_and_nan():
```

Comment:
Why is test operator ? You can actually add it in test_ndarray.py. Actually you can add the following code 
```
allow_nan = np.random.randint(0, 2)
if allow_nan:
    total_nans = np.random.randint(0, dat.size//10+1)
    dat.ravel()[np.random.choice(dat.size, total_nans, replace=False)] = np.nan
```
after this line: https://github.com/apache/incubator-mxnet/blob/72b4d9b8261e0783989447ad78a09e8573aee853/tests/python/unittest/test_ndarray.py#L598
This will randomly decide whether to add Nans to input or not and put nans (roughly 10% of the size of input) inside the input randomly

Reply:
Thank you! I will move the testcase into test_ndarray.py

Reply:
Hi @access2rohit , I have moved the testcase into test_ndarray.py.